src/devices/src/virtio/block/device.rs

```diff
@@ -30,16 +30,39 @@ use super::{
 use crate::virtio::VIRTIO_MMIO_INT_CONFIG;
 use crate::Error as DeviceError;
 
+/// Configuration options for disk caching.
+#[derive(Clone, Copy)]
+pub enum CacheType {
+    /// Flushing mechanic will be advertised to the guest driver, but
+    /// the operation will be a noop.
+    Unsafe,
+    /// Flushing mechanic will be advertised to the guest driver and
+    /// flush requests coming from the guest will be performed using
+    /// `fsync`.
+    Writethrough,
```

Comment:
`writethrough` is not the best name since that usually means **no host FS caching** which is not the case here.

Unfortunately I don't have a better name right now 😄 Suggestions are welcome!

Reply:
I think the right naming is `writeback`. This is the term QEMU uses as well: https://documentation.suse.com/sles/11-SP4/html/SLES-kvm4zseries/cha-qemu-cachemodes.html#id-1.10.6.6.5

Reply:
`writeback` is indeed the right name. I copy-pasted the wrong one the first time around 😆. Nice catch.